exporter/exporterhelper/internal/base_exporter.go

```diff
@@ -70,7 +70,7 @@ func NewBaseExporter(set exporter.Settings, signal pipeline.Signal, options ...O
 	}
 
 	//nolint: staticcheck
-	if be.batcherCfg.MinSizeItems != nil || be.batcherCfg.MaxSizeItems != nil {
+	if be.batcherCfg.MinSizeItems != 0 || be.batcherCfg.MaxSizeItems != 0 {
```

Comment:
Potentially, we could update the `Unmarshal` func to set a private field indicating whether `min/max_size_items` were set, but that would require moving/aliasing the `exporterbatcher.Config` in some internal module along with a function showing this warning. I could do that separately.